Fix for installation stopping due to missing documentation.

Henrik R. Nagel, 20190912
diff -ruN ParaView-v5.5.2.orig/Plugins/StreamLinesRepresentation/doc/index.html ParaView-v5.5.2/Plugins/StreamLinesRepresentation/doc/index.html
--- ParaView-v5.5.2.orig/Plugins/StreamLinesRepresentation/doc/index.html	1970-01-01 01:00:00.000000000 +0100
+++ ParaView-v5.5.2/Plugins/StreamLinesRepresentation/doc/index.html	2019-08-23 13:52:03.000000000 +0200
@@ -0,0 +1,14 @@
+<html>
+  <head>
+    <title>Stream Lines Representation</title>
+  </head>
+  <body>
+    <h1>Stream Lines Representation Plugin</h1>
+
+    <div>
+    This plugin adds support for <i>Stream Lines Representation</i> to
+    <b>ParaView</b>.
+    <div>
+
+  </body>
+</html>
